uiowa/uids
# Feature: Admissions viewbook

# How to test

- Verify that https://uids.brand.uiowa.edu/branches/feature_admissions_viewbook/components/detail/viewbook.html is mostly functional (cta icons are missing)
- Verify that `Aerial` and `Grid panel` components have been moved within a `Viewbook` directory
- Review changes on https://github.com/uiowa/uids/pull/292/files
